packages/suite/src/utils/suite/sentry.ts

```diff
@@ -18,3 +19,19 @@ export const unsetSentryUser = () => {
         scope.setUser(null);
     });
 };
+
+export const setSentryTransport = (transportEvent: TransportEvent) => {
+    if (transportEvent.type === 'transport-start') {
+        const { type, version } = transportEvent.payload;
+        Sentry.configureScope(scope => {
+            // todo: probably merge with already existing context? does it have callback fn sig?
```

Comment:
sentry do that automatically, just the name `setContext` is confusing (it's more like `addContext`)